10_construct-database/13_construct_county_race-share-geography.qmd

```diff
@@ -87,27 +87,27 @@ bind_rows(
 # Correct subgroup category
 
 read_csv(here("data", "temp", "county_debt_longitudinal_race-ethnicity.csv")) |>
-  mutate(subgroup = case_when(subgroup == "Majority White" ~ "Majority White, Non-Hispanic",
-                              subgroup == "No Majority Race/Ethnicity" ~ "Mixed Race and Ethnicity", 
-                              subgroup == "Majority Non-White" ~ "Majority Households of Color",
+  mutate(subgroup = case_when(subgroup == "Majority White" ~ "White communities",
+                              subgroup == "No Majority Race/Ethnicity" ~ "Mixed communities", 
+                              subgroup == "Majority Non-White" ~ "Communities of color",
                               TRUE ~ subgroup),
          subgroup_type = ifelse(subgroup == "All", "all", subgroup_type)) |>
   write_csv(
     here("data", "temp", "county_debt_longitudinal_subgroup_race_share.csv")
   )
 
 read_csv(here("06_neighborhoods/environment/data/final/environment_county_race-ethnicity_longitudinal.csv")) |>
-  mutate(subgroup = case_when(subgroup == "No Majority Race/Ethnicity" ~ "Mixed Race and Ethnicity", 
-                              subgroup == "Majority Non-White" ~ "Majority Households of Color",
+  mutate(subgroup = case_when(subgroup == "No Majority Race/Ethnicity" ~ "Mixed communities", 
+                              subgroup == "Majority Non-White" ~ "Communities of color",
                               TRUE ~ subgroup)) |>
```

Comment:
I think here you will have to add a line of code for White communities - I did not have to include Majority White, Non-Hispanic in the case_when because it was already written that way.